deploy/packer/base.json

```diff
@@ -1,12 +1,41 @@
 {
   "description": "taskcluster worker system dependencies",
   "variables": {
-    "workerRevision": ""
+    "papertrail":             "",
+    "privateKeyLocation":     "",
+    "sslCertificateLocation": "",
+    "sslKeyLocation":         "",
+    "workerRevision":         ""
   },
   "provisioners": [
+    { "type": "file",
+      "source": "{{user `privateKeyLocation`}}",
+      "destination": "/tmp/docker-worker-priv.pem"
+    },
+    { "type": "file",
+      "source": "{{user `sslCertificateLocation`}}",
+      "destination": "/tmp/star_taskcluster-worker_net.crt"
+    },
+    { "type": "file",
+      "source": "{{user `sslKeyLocation`}}",
+      "destination": "/tmp/star_taskcluster-worker_net.key"
```

Comment:
I'll leave it for now, because if we did deicde to deploy outside of TC someday ( I hope!) there is a bunch of other work needed to be done to the worker to make that a reality.  Best to just tackle it all at once.